### PR TITLE
CDPCAM-83 Add cross-account role actions validation for Azure

### PR DESCRIPTION
**Pull request by mahendra-rk**
Signed-off-by: Mahendra Korepu <mkorepu@cloudera.com>

### CODE REVIEW

File: cdpctl/validation/infra/validate_azure_cross_account_identity.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+
+@pytest.mark.azure
+@pytest.mark.infra
+def azure_cross_account_identity_exists_validation(
+    config: Dict[str, Any],
+    auth_client: AuthorizationManagementClient,
+    resource_client: ResourceManagementClient,
+) -> None:  # pragma: no cover
+    """Azure Cross Account Identity exists."""
```
Comment (wonderslug) [inline cdpctl/validation/infra/validate_azure_cross_account_identity.py:89]: Lets maybe change this to "Cloudera Management Console Operator Identity" to match close to the documentation? At least for what we show the customer.

**Review by wonderslug — CHANGES_REQUESTED** [review]

File: cdpctl/validation/infra/validate_azure_cross_account_identity.py (hunk 1 above)
Comment (wonderslug) [reply cdpctl/validation/infra/validate_azure_cross_account_identity.py:89]: As well any error or warning would need to show it.

File: cdpctl/validation/infra/validate_azure_cross_account_identity.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+
+@pytest.mark.azure
+@pytest.mark.infra
+def azure_cross_account_identity_exists_validation(
+    config: Dict[str, Any],
+    auth_client: AuthorizationManagementClient,
+    resource_client: ResourceManagementClient,
+) -> None:  # pragma: no cover
+    """Azure Cloudera Management Console Operator Identity Identity exists."""
```
Comment (wonderslug) [inline cdpctl/validation/infra/validate_azure_cross_account_identity.py:93]: Duplicate Identity

**Review by wonderslug — CHANGES_REQUESTED** [review]

File: cdpctl/validation/infra/validate_azure_cross_account_identity.py (hunk 2 above)
Comment (mahendra-rk) [reply cdpctl/validation/infra/validate_azure_cross_account_identity.py]: fixed. thanks !

**Review by wonderslug — COMMENTED** [review]
LGTM

**Review by wonderslug — APPROVED** [review]

**Pull request merged**